Tasks/Xcode/xcode.ts

```diff
@@ -319,28 +297,41 @@ async function run() {
                         tl.tool(plist).arg(['-c', 'Add teamID string ' + exportTeamId, exportOptionsPlist]).execSync();
                     }
 
-                    if (xcodeVersion >= 9 && !automaticSigningWithXcode && exportOptions === 'auto') {
-                        // Xcode 9 manual signing, set code sign style = manual
-                        tl.tool(plist).arg(['-c', 'Add signingStyle string ' + 'manual', exportOptionsPlist]).execSync();
+                    if (xcodeVersion >= 9 && exportOptions === 'auto') {
+                        var signStyleForExport = signStyle;
```

Comment:
`let` over `var`?

Reply:
Switched completely to lets in all three tasks.